src/general_manager/apps.py

```
@@ -9,14 +10,76 @@
+        self.initializeGeneralManagerClasses()
+        if getattr(settings, "AUTOCREATE_GRAPHQL", False):
+            self.handleGraphQL()
+
+    def handleReadOnlyInterface(self):
+        self.patchReadOnlyInterfaceSync(GeneralManagerMeta.read_only_classes)
+
+        logger.debug("starting to register ReadOnlyInterface schema warnings...")
+        for general_manager_class in GeneralManagerMeta.read_only_classes:
+            read_only_interface: ReadOnlyInterface = general_manager_class.Interface  # type: ignore
+            register(
+                lambda app_configs, model=read_only_interface._model, manager_class=general_manager_class, **kwargs: read_only_interface.ensureSchemaIsUpToDate(
```

Ruff (0.11.9)

Function definition does not bind loop variable `read_only_interface`

(B023)